### ISSUE TITLE
Lost and not receiving funds

### ISSUE THREAD

**Issue by mlpotesak** [monero-project/monero-gui#4128]
I’m having a problem with my wallet. I tried to receive funds from my miner group to my wallet..I had some problems and after I tried to fix the transfer issue all the funds from my wallet were gone. My wallet is empty. Can you advise?

**Comment by selsta** [comment]
Please open your wallet and go to Settings -> Info and post

- Version
- Wallet mode
- Wallet restore height

**Comment by mlpotesak** [comment]
Version o.18.1.2-unknown 
Advanced mode (Remote node)
Wallet reset height 3

**Comment by plowsof** [comment]
is your wallet fully synchronised? 
valid remote node? try selsta2.featherwallet.net:18081 or node2.monerodevs.org:18089

**Comment by selsta** [comment]
@mlpotesak Can you post the exact text it says in the bottom left corner?

**Comment by mlpotesak** [comment]
Wallet is synchronized
Daememon is synchronized(2841979)
Network status remote node
I hope this is what you need

**Comment by selsta** [comment]
When was the first time you received a transaction into this wallet approximately? Which months / year?

**Comment by mlpotesak** [comment]
2022-07-10

**Comment by mlpotesak** [comment]
Is it important how to enter the date because my full amount is not being showed.
Thanks to everyone For your help.

**Comment by selsta** [comment]
Yes, you have to enter it the same way I wrote in my previous comment.

What is the current value of Wallet Restore height?

**Comment by selsta** [comment]
Glad to hear :)
